web/apps/labelstudio/src/components/Form/Elements/Select/Select.scss

```diff
@@ -35,6 +36,8 @@
 select:disabled {
   background: var(--sand_200);
   color: var(--sand_500);
-  font-family: Figtree, sans-serif;
+
+  --font-family: figtree, sans-serif;
```

Comment:
```suggestion
```
I'd check if this is needed at all, seems like a holdover but possibly was needed to be set directly to override antd.